src/redux/categories/categories.js

```diff
@@ -0,0 +1,24 @@
+const GATEGORY_STATUS = 'BookStore/categries/GATEGORY_STATUS';
+
+export const checkStatus = () => ({
+  type: GATEGORY_STATUS,
+});
+
+const initialState = {
+  categories: [],
+  status: '',
+};
+
+const categoriesReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case GATEGORY_STATUS:
+      return {
+        categories: [],
+        status: 'Under construction',
+      };
+
+    default:
+      return state;
+  }
+};
```

Comment:
- [ ] As per the requirements, kindly make sure to display "Under Construction" when the Check status button is clicked.

https://user-images.githubusercontent.com/70269507/210616018-e045b40d-a37e-44a3-8b01-c700199f27bd.mov